src/TodoList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Todo from './Todo';
+
+function TodoList({ todoList, onClick }) {
+  return (
+    <div>
+      {todoList.length ? (
+        <ol>
+          {todoList.map((todo) => (
+            <Todo key={todo.id} todo={todo} onClick={onClick} />
```

Comment:
```suggestion
            <Todo 
              key={todo.id} 
              todo={todo} 
              onClick={onClick}
            />
```

props를 3개 정도 넘기면 보통은 줄바꿈을 해서 가독성을 좋게합니다.